include/openssl/ocsp.h

```diff
@@ -58,6 +62,9 @@ extern "C" {
 // Certificates included within |bs| or |req| will be included in the
 // search for the signing certificate by default, unless |OCSP_NOINTERN| is set.
 #define OCSP_NOINTERN 0x2
+// OCSP_NOSIGS does nothing. In OpenSSL, this skips signature verification in
+// |OCSP_basic_verify| and |OCSP_request_verify|.
```

Comment:
so we'll _always_ check the signatures, right?

Reply:
Yes, the intention was to not do this until it's specifically asked for. There's not really a scenario where the signature shouldn't be verified.